parachain/precompiles/vesting/VestingInterface.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: GPL-3.0-only
```

Comment:
Is it possible to add a test case in parachain ts-tests?

Reply:
Sure. I will add one.